scripts/releaseSteps.js

```diff
@@ -115,6 +121,20 @@ async function createGitHubRelease({ newVersion, releaseNotes }) {
   return { releaseId, htmlUrl, uploadUrl };
 }
 
+async function cleanSource() {
+  packages.map(async (packageName) => {
```

Comment:
Does this need a Promise.all to run these in parallel?

Reply:
@dangerismycat Good spotting - this didn't need to be `async` - I made it sync instead